src/Tooltip/Tooltip.component.tsx

```diff
@@ -121,37 +155,48 @@ export class Tooltip extends React.Component<
 
     render(): React.ReactElement<Tooltip> {
         const {
-            className,
+            arrowIndent,
+            arrowSize,
+            background = 'rgba(0, 0, 0, 0.8)',
+            border,
             children,
+            className,
+            color,
             content,
+            delay,
+            hidden : propsHidden,
+            maxWidth,
             position = 'topEnd',
+            shadow,
+            showArrow,
             wrapContent,
-            background,
-            color,
-            maxWidth,
-            delay,
             ...props
         } = this.props;
         const {
             height,
             width,
-            hidden,
+            hidden : stateHidden,
             toolTipHeight,
             toolTipWidth,
         } = this.state;
+        // If the dev passes hidden as a prop then Tooltip's behavior changes, and whether or not it
+        // is visible is controlled by that prop and not by state.
+        const hiddenOveride = propsHidden !== undefined;
```

Comment:
I'm uncertain about this approach. I feel like it makes logical sense, but it feels somewhat tacked on.

Reply:
I think this is fine, but I would probably invert the logic to a `show` or `open` prop instead. It wouldn't be obvious to me that `hidden={false}` opens it up.

Reply:
I like that better. I had a feeling that maybe it was just the wording of the prop that was bugging me.